services/ffc/ffc-demo/ffc-demo-payment-service/infra/snd/01/patch.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   chart:
     spec:
       chart: ffc-demo-payment-service-infra
-      version: "4.2.24" # {"$imagepolicy": "flux-config:ffc-demo-payment-service-snd-01:tag"}
+      version: "0.1.0" # {"$imagepolicy": "flux-config:ffc-demo-payment-service-snd-01:tag"}
```

Comment:
all of these shouldnt be reverted?

Reply:
@DaRoszko  yes. We discussed this with @aadev1  and @jeevankuduvaravindran . Asad is updating the script to do incremental updates, leaving current version as is if greater than 0.1.0